server/repository/assignments.go

```diff
@@ -152,3 +154,27 @@ func (repo *Assignments) Update(assignmentID int, answer string, duration int) e
 
 	return err
 }
+
+// CountUserAssigments returns number of assigments in given experiment for the given user
+func (repo *Assignments) CountUserAssigments(experimentID, userID int) (int, error) {
```

Comment:
Typo in several places, Assigment -> assignment

Reply:
thank you! I never spell it correctly